packages/webawesome/src/components/button/button.ts

```diff
@@ -135,24 +134,23 @@ export default class WaButton extends WebAwesomeFormAssociatedElement {
 
   private constructLightDOMButton() {
     const button = document.createElement('button');
+
+    for (const attribute of this.attributes) {
+      button.setAttribute(attribute.name, attribute.value);
+    }
```

Comment:
I'm fine with this, but I'm curious if we were missing any attributes in the array below.